sconstruct

```diff
@@ -61,6 +61,7 @@ keyCommandsLangBlacklist=set([])
 
 vars = Variables()
 vars.Add("version", "The version of this build", versionInfo.version)
+vars.Add("version_build", "A unique number for this build.", "0")

```

Comment:
Is there a reason you went with version_build rather than the camel case versionBuild? It just seems out of place because all the other variables are camel case. I'm guessing you're trying to make it clear that build is a component of version...? Same for the names in the versionInfo module.